tests/pfcwd/test_pfcwd_function.py

```diff
@@ -873,6 +873,9 @@ def test_pfcwd_actions(self, request, fake_storm, setup_pfc_test, setup_dut_test
         self.timers = setup_info['pfc_timers']
         self.ports = setup_info['selected_test_ports']
         self.test_ports_info = setup_info['test_ports']
+        if self.dut.topo_type == 't2':
+            key, value = list(self.ports.items())[0]
```

Comment:
Could you share more details about the change, why only use the first pair? Follow the previous behavior? thanks